LICENSE.md

```diff
@@ -6,115 +6,113 @@ Creative Commons Corporation (“Creative Commons”) is not a law firm and does
 
 Creative Commons public licenses provide a standard set of terms and conditions that creators and other rights holders may use to share original works of authorship and other material subject to copyright and certain other rights specified in the public license below. The following considerations are for informational purposes only, are not exhaustive, and do not form part of our licenses.
 
-* __Considerations for licensors:__ Our public licenses are intended for use by those authorized to give the public permission to use material in ways otherwise restricted by copyright and certain other rights. Our licenses are irrevocable. Licensors should read and understand the terms and conditions of the license they choose before applying it. Licensors should also secure all rights necessary before applying our licenses so that the public can reuse the material as expected. Licensors should clearly mark any material not subject to the license. This includes other CC-licensed material, or material used under an exception or limitation to copyright. [More considerations for licensors](http://wiki.creativecommons.org/Considerations_for_licensors_and_licensees#Considerations_for_licensors).
```

Comment:
Not sure how I feel about the license getting hit by prettier - this shouldn't be happening right?

Reply:
I think it's fine - all it's actually changing is `__` to `**` and `___` to `**_` / `_**` ([equivalent styling from the Markdown docs](https://www.markdownguide.org/basic-syntax/#emphasis)) along with some whitespacing. Once again just keeping things consistent